beacon_node/src/config.rs

```diff
@@ -747,19 +747,22 @@ pub fn get_config<E: EthSpec>(
     }
 
     if cli_args.is_present("disable-proposer-reorgs") {
-        client_config.chain.re_org_threshold = None;
+        client_config.chain.re_org_head_threshold = None;
+        client_config.chain.re_org_parent_threshold = Some(ReOrgThreshold(u64::MAX));
     } else {
-        client_config.chain.re_org_threshold = Some(
+        client_config.chain.re_org_head_threshold = Some(
             clap_utils::parse_optional(cli_args, "proposer-reorg-threshold")?
                 .map(ReOrgThreshold)
-                .unwrap_or(DEFAULT_RE_ORG_THRESHOLD),
+                .unwrap_or(DEFAULT_RE_ORG_HEAD_THRESHOLD),
         );
         client_config.chain.re_org_max_epochs_since_finalization =
             clap_utils::parse_optional(cli_args, "proposer-reorg-epochs-since-finalization")?
                 .unwrap_or(DEFAULT_RE_ORG_MAX_EPOCHS_SINCE_FINALIZATION);
         client_config.chain.re_org_cutoff_millis =
             clap_utils::parse_optional(cli_args, "proposer-reorg-cutoff")?;
 
+        // TODO(is_parent_strong) do we want re_org_parent_threshold settable?
```

Comment:
IMO yes, as it can be helpful to have some nodes being more aggressive with reorgs, even on networks with low participation like testnets